config/datatypes_conf.xml.sample

```diff
@@ -484,6 +531,13 @@
     <sniffer type="galaxy.datatypes.proteomics:Msp"/>
     <sniffer type="galaxy.datatypes.proteomics:SPLib"/>
     <sniffer type="galaxy.datatypes.proteomics:ThermoRAW"/>
+    <sniffer type="galaxy.datatypes.molecules:SDF"/>
+    <sniffer type="galaxy.datatypes.molecules:PDB"/>
+    <sniffer type="galaxy.datatypes.molecules:MOL2"/>
+    <sniffer type="galaxy.datatypes.molecules:InChI"/>
+    <sniffer type="galaxy.datatypes.molecules:FPS"/>
+    <sniffer type="galaxy.datatypes.molecules:CML"/>
+    <!-- TODO: see molecules.py <sniffer type="galaxy.datatypes.molecules:SMILES"/>-->
```

Comment:
Can you update the doctests for ``guess_ext`` with sniffer tests for these datatypes - at least the ones you have test data in the repo for:

https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/sniff.py#L259